graylog2-web-interface/src/views/components/Query.test.jsx

```diff
@@ -57,14 +64,14 @@ describe('Query', () => {
       },
     };
     const wrapper = mount((
-      <Query results={results}
-             widgetMapping={widgetMapping}
-             widgets={widgets}
-             onToggleMessages={() => {}}
-             queryId="someQueryId"
-             showMessages
-             allFields={Immutable.List()}
-             fields={Immutable.List()} />
+      <SUT results={results}
+           widgetMapping={widgetMapping}
```

Comment:
Can we move all of the props which are constantly reused to the `SUT` definition? This way we can separate the boilerplate and override in tests what's actually different, which makes it way easier to understand.